content/terraform-docs-agents/v1.22.x/docs/cloud-docs/agents/requirements.mdx

```diff
@@ -6,21 +6,33 @@ description: >-
 
 # Agent Requirements
 
-~> Important: All agents in an agent pool should have the same operating system and hardware resources available. If the agents are using different resources, run performance can vary significantly between each agent in the pool.
-
 Ensure your system meets the following requirements before installing and configuring HCP Terraform Agents. Refer to [HCP Terraform Agents on Terraform Enterprise](/terraform/enterprise/admin/agents-on-tfe) for additional Terraform Enterprise requirements.
 
-## Supported Operating Systems
+## Environment
+
+You can deploy agents to x86_64 and ARM64 Linux. You can also run the agent in Docker using our official [HCP Terraform Agent Docker container](https://hub.docker.com/r/hashicorp/tfc-agent).
+
+All agents in an agent pool should have the same operating system. If the agents use different resources, run performance can vary significantly between each agent in the pool.
+
+The architecture of the agent pool is determined by the first agent that connects to Terraform. For example, when the first agent to register to an empty pool is an ARM64 agent, subsequent agents that register must be ARM64 agents. Attempting to register an ARM64 agent to an existing x86-only agent pool results in an error.
```

Comment:
A small language nitpick: is our current guideline to use "Terraform" when talking about both Terraform the CLI tool and HCP Terraform? Agent registration is entirely an HCP Terraform affair, so it may be more accurate to say:

> The architecture of the agent pool is determined by the first agent that connects to HCP Terraform.